cmd/internal/rssshrinker/rssshrinker_linux.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+// Package rssshrinker provides functions to reduce the process’ RSS
+package rssshrinker
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"runtime/debug"
+	"strconv"
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+/*
+#include <malloc.h>
+*/
+import "C"
+
+//revive:disable:var-naming
+
+// MADV_PAGEOUT Reclaim these pages.
+const MADV_PAGEOUT = 21
+
+//revive:enable:var-naming
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I believe this should work similarly while being simpler and less error prone
```suggestion
// MADV_PAGEOUT Reclaim these pages.
//
//nolint:revive
const MADV_PAGEOUT = 21
```